index.js

```diff
@@ -0,0 +1,35 @@
+import { register } from 'pane-registry'
+const MeetingPane = require('./meetingPane.js').default
```

Comment:
Not sure how to avoid having to do the .default here, see also https://github.com/solid/meeting-pane/pull/9/files#diff-718c37bbffddefd179a26a9a9d5e5a49R15-R16

Reply:
Since you're using ES modules, you should be able to just do `import MeetingPane from './meetingPane.js';`.

Reply:
Right, in https://github.com/solid/chat-pane/pull/26/files#diff-d39293588b0c2de1b16cc23c9f4dc0ebR15 that does work. Will fiddle around with it some more. :)

Reply:
Hm, see f0b0b03